tests/test_function.py

```diff
@@ -1363,3 +1363,11 @@ def test_divide(self):
         f = function.Argument('test', shape=(2, 3), dtype=int)
         self.assertIsNot(f / 1, f)
         self.assertIsNot(f / 1., f)
+
+
+class misc(TestCase):
+
+    def test_truthiness(self):
```

Comment:
Suggestion: move this test to `class Array` at the top of the module.

Reply:
done